### PR TITLE
Add buyer, seller and properties field to meeting

### PR DESCRIPTION
**Pull request by thony-ui**

### CODE REVIEW

**Review by Virusrwj223 — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/model/meeting/Meeting.java [hunk 1]
```diff
@@ -31,6 +43,20 @@ public MeetingDate getMeetingDate() {
...
+    public Type getType() {
+        return this.type;
+    }
+
+    public PostalCode getPostalCode() {
+        return this.postalCode;
+    }
+
     /**
      * Returns true if both persons have the same name.
```
Comment (jolwnn) [inline src/main/java/seedu/address/model/meeting/Meeting.java:61]: Reminder to self to fix this haha

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 2]
```diff
@@ -37,22 +37,28 @@
 public class SampleDataUtil {
     public static Person[] getSamplePersons() {
         return new Person[] {
-            new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
+            new Person(new seedu.address.model.person.Name("Alex Yeoh"),
```
Comment (jolwnn) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:40]: what's happening here is this not the old address book stuffs

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 3]
```diff
@@ -37,22 +37,28 @@
...
+                    new Phone("87438807"), new Email("alexyeoh@example.com"),
```
Comment (jolwnn) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:41]: what's happening here is this not the old address book stuffs

Comment (jolwnn) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java:41]: Oh I see is it because of the conflict in client name and person name

**Review by jolwnn — APPROVED** [review]
LGTM as well

**Pull request merged**